packages/Babylonian-Core.package/BPExample.class/instance/assertions.st

```diff
@@ -0,0 +1,6 @@
+accessing
+assertions
+
+	^ self currentTrace
+		ifNil: [#()]
```

Comment:
nitpick: in `traceValues` it's `{}` returned. as you create a read-only one here, could you change it in `traceValues` to `#()` too, so it's more consistent?